packages/libs/src/sdk/api/comments/CommentsAPI.ts

```diff
@@ -14,6 +16,11 @@ import {
 
 import { CommentMetadata } from './types'
 
+type CommentNotificationOptions = OverrideProperties<
```

Comment:
ooh nice, this should help with some bugs like when we accidentally passed in an empty object into metadata